src/feedback/commands/submitFeedback.ts

```diff
@@ -0,0 +1,67 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { _Blob } from 'aws-sdk/clients/lambda'
+import _ = require('lodash')
+import * as vscode from 'vscode'
+import { ext } from '../../shared/extensionGlobals'
+import { ExtensionUtilities } from '../../shared/extensionUtilities'
+import { getLogger, Logger } from '../../shared/logger'
+import { BaseTemplates } from '../../shared/templates/baseTemplates'
+import { FeedbackTemplates } from '../templates/feedbackTemplates'
+import { submitFeedbackListener } from './submitFeedbackListener'
+
+export async function submitFeedback(listener?: (message: any) => Promise<void>) {
+    const logger: Logger = getLogger()
+
+    try {
+        const panel = vscode.window.createWebviewPanel('html', 'Submit Quick Feedback', vscode.ViewColumn.One, {
+            retainContextWhenHidden: true,
```

Comment:
VS Code recommends against this unless we have a strong reason for it. That being said, this SHOULD be a fairly light case.

Have you thought about using get/setState? https://code.visualstudio.com/api/extension-guides/webview#getstate-and-setstate

Reply:
Yes I saw that and was experimenting with get/set state. However, my attempts to save and restore state with vue didn't work. So I decided not to spend more time on it and to go with this because it is indeed light and the user shouldn't really have this running in the background.

Also note that the SAM view also runs in the background and that's probably more impactful. We should definitely look into getting this working.